Source/bmalloc/bmalloc/StaticPerProcess.h

```diff
@@ -81,11 +81,11 @@ class StaticPerProcess {
     {
         using Storage = typename StaticPerProcessStorageTraits<T>::Storage;
         LockHolder lock(Storage::s_mutex);
-        if (!Storage::s_object.load(std::memory_order_consume)) {
+        if (!Storage::s_object.load(std::memory_order_relaxed)) {
             T* t = new (&Storage::s_memory) T(lock);
             Storage::s_object.store(t, std::memory_order_release);
```

Comment:
Yeah, so I'm pretty sure the atomic here is only needed to provide atomicity and not memory order. If getFastCase() fails, it falls back to the slow case, that uses the mutex, and that guarantees memory order. So I'm pretty sure your changes are correct, but also that you should change this line too.

Reply:
Addressed!